dropwizard-app/src/main/java/org/coner/boundary/CompetitionGroupBoundary.java

```diff
@@ -0,0 +1,34 @@
+package org.coner.boundary;
+
+import org.coner.core.domain.CompetitionGroup;
+
+/**
+ *
+ */
+public class CompetitionGroupBoundary extends AbstractBoundary<
```

Comment:
Do you not want to continue the pattern of having a protected constructor with a getInstance() static method for getting the singleton instance of the CompGroupBoundary?